internal/cephfs/store/fsjournal.go

```diff
@@ -128,6 +128,14 @@ func CheckVolExists(ctx context.Context,
 		}
 		err = cloneState.ToError()
 		if errors.Is(err, cerrors.ErrCloneInProgress) {
+			progressReport := cloneState.GetProgressReport()
```

Comment:
it would be good to surface this error to the user as an error message not everyone will have access/check the log message

Reply:
Done

Reply:
i dont see any update to add progressReport to the error message, can you please point me to the code where we are adding/returning progressReport as part of GRPC error message?

Reply:
Sorry, I misunderstood the previous comment.
Updated the code to return the progress report values as part of `err` value of the function call.
Please check now.